Makefile.in

```diff
@@ -195,11 +195,16 @@ ifeq ($(shell uname), Darwin)
 endif
 
 #
-# All translation message catalogs
+# All translation message catalogs, filter files based on LINGUAS.
 #
 TRANSLATIONS_SRC := $(wildcard po/*.po)
-ifdef HAVE_GETTEXT
+ifeq ($(HAVE_GETTEXT), 1)
```

Comment:
that's not explicitly required by that change, I just changed it to be more precise since I was working on that part of the code. 

it seems that 
#### configure.ac
```393:  AC_CHECK_FUNC( gettext, HAVE_GETTEXT=1, HAVE_GETTEXT=0 )```

 sets it to zero, which makes variable non-existent. so there is no difference what to use.

I've tested with simple ```@echo "gettext $(HAVE_GETTEXT)"``` in a Makefile.

I can edit the line 767 to use ifeq or change line 201 back to ifdef, whichever is best for you.